### PR TITLE
fix: Try limit default file watcher 

### PR DESCRIPTION
**Pull request by asafMasa**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✔                                                                        |
| New feature     | ✖                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✖                                                                       |

Try limit default file watcher

### CODE REVIEW

File: MergerService/Program.cs [hunk 1]
```diff
@@ -1,12 +1,25 @@
...
 
-// Add services to the container.
+var builder = WebApplication.CreateBuilder(new WebApplicationOptions
+{
+    ContentRootPath = reflectionLocationDir,
+    WebRootPath = reflectionLocationDir,
+    Args = args
+});
 
+builder.Configuration["hostBuilder:reloadConfigOnChange"] = "false";
```
Comment (asafMasa) [inline MergerService/Program.cs:17]: https://github.com/dotnet/aspnetcore/blob/78ae8a5e69417b3d886a81998289fa8026aeaa91/src/DefaultBuilder/src/WebApplicationBuilder.cs#L271

Comment (asafMasa) [reply MergerService/Program.cs:17]: https://learn.microsoft.com/en-us/aspnet/core/fundamentals/host/generic-host?view=aspnetcore-8.0#disable-app-configuration-reload-on-change

File: MergerService/Program.cs [hunk 2]
```diff
@@ -1,12 +1,25 @@
...
 using MergerService.Src;
 using MergerService.Utils;
+using System.Reflection;
 
-var builder = WebApplication.CreateBuilder(args);
+string reflectionLocation = Assembly.GetExecutingAssembly().Location;
+string reflectionLocationDir = Path.GetDirectoryName(reflectionLocation);
 
-// Add services to the container.
+var builder = WebApplication.CreateBuilder(new WebApplicationOptions
```
Comment (asafMasa) [inline MergerService/Program.cs:10]: https://stackoverflow.com/questions/69909593/asp-net-6-custom-webapplicationfactory-throws-exception

File: MergerService/Program.cs (hunk 1 above)
Comment (asafMasa) [reply MergerService/Program.cs:17]: [appSettingsPath](https://github.com/dotnet/extensions/issues/2939)

**Review by shimoncohen — APPROVED** [review]

**Pull request merged**